templates/hooks/table/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,6 @@
+	if tableDeleting(r) {
+		return requeueWaitWhileDeleting
+	}
```

Comment:
Table could be in deleting state before calling delete? I imagine if someone messed with it in the console (outside ACK)?

Reply:
Yup - not sure if this is something we should consider at this stage.